i18n/messages/nl.json

```
@@ -885,6 +900,11 @@
     "description": "Tooltip for 'includePartners' builder field",
     "originalDefault": "Whether to add partners information to the component."
   },
+  "ZRM6J8": {
+    "defaultMessage": "Mogelijke kaartmateriaal interacties",
```

```suggestion
    "defaultMessage": "Mogelijke kaartmateriaalinteracties",
```
single samengesteld woord